src/main.js

```diff
@@ -1,18 +1,15 @@
 import Cycle from '@cycle/core';
 import CycleWeb from '@cycle/web';
 
-import {scrollTopBehavior} from './values';
-
-import MainView from './views/main-view';
+import intent from './intent';
+import model from './models/main-model';
+import view from './views/main-view';
 
 function main({DOM}) {
-  var scrollTableNodeStream = DOM.get('#scroll-table-container', 'scroll').subscribe(
-    e => scrollTopBehavior.onNext(e.srcElement.scrollTop)
-  );
```

Comment:
ah, this was definitely misnamed. and yeah, since i was just updating the subject with a new value, i was using onNext here, but doesn't make sense without it anymore.